Examples/SmokeTest/SmokeTest/ContentView.swift

```diff
@@ -78,6 +86,10 @@ struct ContentView: View {
                         systemImage: "paintpalette"
                     )
                 }
+
+            NavigationExamplesView()
+                .padding()
+                .tabItem { Label("Navigation", systemImage: "globe") }
```

Comment:
Do we want to add an `onAppear` to this one like the others?

Reply:
whoops yeah. Love those logical merge conflicts.

Reply:
Aha! This is kinda subtle. The UIKit view did get added after the fact and needs an onAppear 

This one, though, doesn't need it, because, the nested NavigationStack / NavigationSplitView will do it for us.